upstairs/src/lib.rs

```diff
@@ -4128,7 +4129,21 @@ impl Downstairs {
             }
         }
 
-        if !successful_hash {
+        if let Some(valid_hash) = valid_hash {
```

Comment:
Since you are here already, could we correct the comment up at line 4111, or at least mention that
it is only for the SQL backend? Thanks!

Reply:
Rgr, done in 5d46ff4